src/fleet/azext_fleet/commands.py

```diff
@@ -118,3 +125,12 @@ def load_command_table(self, _):
         g.custom_show_command("show", "show_gate")
         g.custom_command("update", "update_gate")
         g.custom_command("approve", "approve_gate")
+
+    # fleet namespaces command group
+    with self.command_group("fleet namespace", managed_namespaces_sdk, client_factory=cf_managed_namespaces) as g:
```

Comment:
set `is_preview=True` (maybe here, not quite sure) to indicate that this is preview functionality

(please also ask David if the same was missed for gates?)